src/iminuit/repr_html.py

```diff
@@ -30,13 +30,13 @@ def tag(name, *args, **kwargs):
     head = "<" + name
     for k in sorted(kwargs):
         v = kwargs[k]
-        head += ' %s="%s"' % (k, v)
+        head += f' {k}="{v}"'
     head += ">"
     tail = "</%s>" % name
     if len(args) == 0:
         return [head + tail]
     if len(args) == 1 and isinstance(args[0], str):
-        return ["{0} {1} {2}".format(head, args[0], tail)]
+        return ["{} {} {}".format(head, args[0], tail)]
```

Comment:
pyupgrade tried to be very fugal with f-strings, only using them when it's clearly better - if there are non-simple expressions, like `args[0]`, it's won't automatically suggest they be used (though some of those cases, I still think the f-string would be better).